internal/nginx/config/servers.go

```diff
@@ -314,3 +316,16 @@ func createDefaultRootLocation() http.Location {
 		Return: &http.Return{Code: http.StatusNotFound},
 	}
 }
+
+func formatPathType(pathType v1beta1.PathMatchType) string {
+	switch pathType {
+	case v1beta1.PathMatchPathPrefix:
+		return "prefix"
+	case v1beta1.PathMatchExact:
+		return "exact"
+	case v1beta1.PathMatchRegularExpression:
```

Comment:
Yeah I could do that

Reply:
We're okay using panic as a proper way to error?

Reply:
Yes, when it's an exceptional situation, that should not occur. I think that applies here, but I'll ask @pleshakov to weigh in.

Reply:
Do we need the third case? The default case will catch path type regex.

Reply:
Not technically, no. Though it would future proof our implementation so when regex support is added, this isn't missed. But I can remove if that's preferred.

Reply:
Personally, I prefer the simpler switch statement without the regex case. The panic will alert us when we add regex support.

Reply:
Yep, good point. I'll remove it.